docs/build-your-first-basic-workflow/build-your-first-workflow.mdx

```diff
@@ -0,0 +1,2101 @@
+---
+id: build-your-first-workflow
+title: Build Your First Workflow
+sidebar_label: "Part 1: Build Your First Workflow"
+hide_table_of_contents: true
+description: Learn Temporal's core concepts by building a money transfer Workflow. Experience reliability, failure handling, and live debugging in a short tutorial.
+keywords:
+  - temporal
+  - workflow
+  - tutorial
+  - money transfer
+  - reliability
+tags:
+  - Getting Started
+  - Tutorial
+---
+
+import { SetupSteps, SetupStep, CodeSnippet } from "@site/src/components/elements/SetupSteps";
+import { CallToAction } from "@site/src/components/elements/CallToAction";
+import { TemporalProgress } from "@site/src/components/TemporalProgress";
+import { StatusIndicators } from "@site/src/components/StatusIndicators";
+import { RetryPolicyComparison } from "@site/src/components/RetryPolicyComparison";
+import { NextButton } from "@site/src/components/TutorialNavigation";
+import SdkTabs from "@site/src/components/elements/SdkTabs";
+import { FaPython, FaJava } from 'react-icons/fa';
+import { SiGo, SiTypescript, SiPhp, SiDotnet, SiRuby } from 'react-icons/si';
+
+export const TUTORIAL_LANGUAGE_ORDER = [
+  { key: 'py', label: 'Python', icon: FaPython },
+  { key: 'go', label: 'Go', icon: SiGo },
+  { key: 'java', label: 'Java', icon: FaJava },
+  { key: 'ts', label: 'TypeScript', icon: SiTypescript },
+  { key: 'php', label: 'PHP', icon: SiPhp },
+  { key: 'dotnet', label: '.NET', icon: SiDotnet },
+  { key: 'rb', label: 'Ruby', icon: SiRuby },
+];
+
+In this tutorial, you'll build and run your first Temporal application. 
+You'll understand the core building blocks of Temporal and learn how Temporal helps you build crash proof applications through durable execution. 
+
+
+<div style={{display: 'flex', gap: '0.5rem', marginBottom: '2rem'}}>
+  <span className="badge badge--success">Temporal beginner</span>
+</div>
+
+<TemporalProgress steps={[
+  { id: 'part1', label: 'Part 1: Build Your First Workflow', status: 'active' },
+  { id: 'part2', label: 'Part 2: Failure Simulation', status: 'pending' }
+]} />
+
+
+## Introduction
+
+
+### Prerequisites
+
+Before starting this tutorial:
+
+- **Set up a local development environment** for developing Temporal applications
```

Comment:
I feel like the prerequisites are a little unclear to those who are new to our site. I know that if users go through the quick starts if they are set, but if I'm a new user, it's not clear if I am supposed to do the setup AND the quickstarts. 

If we don't want to create a separate environment setup page, then we should just remove the two bullets and just tell them to do the quickstarts as a prerequisite.